cmd/arrai/yaml.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/arr-ai/arrai/rel"
+	"github.com/arr-ai/arrai/translate"
+	"github.com/urfave/cli/v2"
+)
+
+var yamlCommand = &cli.Command{
```

Comment:
I don’t know how useful it is to have a command to translate yaml to arr.ai syntax. In practice, I think the intent will almost always be to do something with the translated values rather than just output them to stdout. On that note, `//encoding.yaml.decode` is certainly useful. In fact, `arrai yx` is easily expressible as `arrai eval '//encoding.yaml.decode(//os.stdin)'`.

To address the obvious question, I have similar reservations about the `jx` command. We don't it either.

Reply:
Agree. Removed yx, kept decode.